src/modules/shared/components/Hero/Hero.types.ts

```diff
@@ -8,7 +8,7 @@ export interface HeroProps {
 	description: string;
 	link: ComponentLink;
 	image: HeroImage;
-	user?: UserSchema;
+	user?: UserSchema | null;
```

Comment:
`?:` + `| null` seems redundant

Reply:
een prop mag zowel `null` als `undefined` zijn:
* `undefined` -> prop hoeft niet aanwezig te zijn
* `null` -> prop is aanwezig maar waarde is null